cmd/snap/cmd_info.go

```diff
@@ -503,6 +503,20 @@ func (iw *infoWriter) maybePrintNotes() {
 	return
 }
 
+func (iw *infoWriter) maybePrintCohortKey() {
+	if !iw.verbose {
+		return
+	}
+	if iw.localSnap == nil {
+		return
+	}
+	if iw.localSnap.CohortKey == "" {
+		return
+	}
+	// 15 is 1 + the length of "refresh-date: "
+	fmt.Fprintf(iw, "cohort:\t%s\n", strutil.ElliptRight(iw.localSnap.CohortKey, iw.termWidth-15))
```

Comment:
I don't think truncating the information here is acceptable in general. Can we do something in general? can we not truncate at least if the output is not a tty?

Reply:
In general no because the cohort is ridiculously long and will wrap on any terminal.
When the output is not a tty, sure. I'll add code for that.